### ISSUE TITLE
ERROR: for unbound-mailcow  Container "4d308fc148e0" is unhealthy.

### ISSUE THREAD

**Issue by copenhaus** [mailcow/mailcow-dockerized#798]
Hi, trying to setup mailcow as a  newbit.

following the script found here:  https://www.maketecheasier.com/setup-email-server-ubuntu/

when I got to "sudo docker-compose up -d", it showed me this error: ERROR: for unbound-mailcow  Container "4d308fc148e0" is unhealthy.

when I ran "docker logs 4d308fc148e0", it showed the following 15 times:
error: database is uninitialized and password option is not specified
  You need to specify one of MYSQL_ROOT_PASSWORD, MYSQL_ALLOW_EMPTY_PASSWORD and MYSQL_RANDOM_ROOT_PASSWORD

thoughts?

thanks in advance.

**Comment by copenhaus** [comment]
OK, i've disabled the health check, now when I run it, I got a bunch of "WARNING" messages, but no major errors.

WARNING: The DBROOT variable is not set. Defaulting to a blank string.
WARNING: The DBNAME variable is not set. Defaulting to a blank string.
WARNING: The DBUSER variable is not set. Defaulting to a blank string.
WARNING: The DBPASS variable is not set. Defaulting to a blank string.
WARNING: The MAILCOW_HOSTNAME variable is not set. Defaulting to a blank string.
WARNING: The WATCHDOG_NOTIFY_EMAIL variable is not set. Defaulting to a blank string.
WARNING: The TZ variable is not set. Defaulting to a blank string.
WARNING: The ADDITIONAL_SAN variable is not set. Defaulting to a blank string.

HOWEVER, when I go to https://<ip address>, I got "502 Bad Gateway"...

Thoughts on this?

Thanks...
